packages/gatsby-theme-newrelic/src/components/Button.js

```diff
@@ -134,11 +139,13 @@ const Button = styled.button`
 
```

Comment:
so we would add a little more code here. We would need to render the button and then apply all the classes (right now it just returns the styled button, but since you are providing an icon you will need to render that within the button. So for example, you will need something like this 

```js 
const Button = ({variant, size, disabled, icon, children}) => (
<button css={css`/*all the styles here*/`}>
    { icon ? <Icon name={icon}/> : null } 
    { children }
</button>
)
```

Reply:
this would replace the styled.div above